certbot/setup.py

```diff
@@ -87,16 +84,22 @@ def read_file(filename, encoding='utf8'):
     'sphinx_rtd_theme',
 ]
 
+# Tools like pip, wheel, and tox are listed here to ensure they are properly
```

Comment:
Could we just (inaccurately) stick all of the dev extras into `test_extras` to get dev dep pinning?

Reply:
We cannot. If we do that, we have problems with dependency conflicts between things like bleeding edge `poetry` and some of the dependencies in our oldest tests.

Reply:
Oh no. Very sad. Ah well.